modules/learn/pages/data/data.adoc

```diff
@@ -24,8 +24,11 @@ Each key:
 
 * Must be a UTF-8 string with no spaces.
 Special characters, such as `(`, `%`, `/`, `"`, and `_`, are acceptable.
-* May be no longer than 250 bytes.
-* Must be unique within its bucket.
+* May be no longer than 246 bytes.
++
+NOTE: For compatibility with earlier versions of Couchbase, the key string can be 250 bytes for the default collection.
```

Comment:
It needs to be made clearer what that compatibility means to the user.  In this case, if they are accessing the default collection 250 bytes is the maximum, they could be accessing the default collection because of a 'legacy' SDK or they could just be explicitly using that particular collection.